SQLCli Release 0.0.68
SQL> loaddriver C:\Users\耿晨雨\Desktop\linkoopdb-jdbc-2.2.3.jar
Driver loaded.
SQL> connect admin/123456
Database connected.
SQL> start LDB_EXT_External_Table/LDBX022-08 create external table by parquet_hive.sql
SQL> drop table t_external_parquet  IF  EXISTS;
0 rows affected
SQL> drop table t_external_hive  IF  EXISTS;
0 rows affected
SQL> drop table t_external_mysql  IF  EXISTS;
0 rows affected
SQL> CREATE EXTERNAL TABLE t_external_parquet BY PARQUET('location':'hdfs://node73:8020/node74/stream74/linkoopdb/data/parquet');
java.sql.SQLException: java.net.ConnectException: Connection refused Caused by [Connection refused] in statement [CREATE EXTERNAL TABLE t_external_parquet BY PARQUET('location':'hdfs://node73:8020/node74/stream74/linkoopdb/data/parquet')]
SQL> select * from t_external_parquet order by id;
java.sql.SQLSyntaxErrorException: user lacks privilege or object not found: T_EXTERNAL_PARQUET in statement [select * from t_external_parquet order by id]
SQL> CREATE EXTERNAL TABLE t_external_hive BY HIVE('table':'studio_test.TEST_TAB');
java.sql.SQLException: java.net.ConnectException: Connection refused Caused by [Connection refused] in statement [CREATE EXTERNAL TABLE t_external_hive BY HIVE('table':'studio_test.TEST_TAB')]
SQL> select * from t_external_hive order by id;
java.sql.SQLSyntaxErrorException: user lacks privilege or object not found: T_EXTERNAL_HIVE in statement [select * from t_external_hive order by id]
SQL> CREATE EXTERNAL TABLE t_external_mysql BY
   > jdbc('table':'EMP',
   > 'driver':'com.mysql.jdbc.Driver',
   > 'url':'jdbc:mysql://192.168.1.72:3306/studio_test',
   > 'user':'test',
   > 'password':'123456');
java.sql.SQLException: java.net.ConnectException: Connection refused Caused by [Connection refused] in statement [CREATE EXTERNAL TABLE t_external_mysql BY
jdbc('table':'EMP',
'driver':'com.mysql.jdbc.Driver',
'url':'jdbc:mysql://192.168.1.72:3306/studio_test',
'user':'test',
'password':'123456')]
SQL> select * from t_external_mysql order by empno;
java.sql.SQLSyntaxErrorException: user lacks privilege or object not found: T_EXTERNAL_MYSQL in statement [select * from t_external_mysql order by empno]
SQL> exit
Disconnected.
